compiler/optimizer/BitVectorAnalysis.cpp

```diff
@@ -1249,18 +1240,23 @@ template<class Container>bool TR_ForwardDFSetAnalysis<Container *>::analyzeNodeI
            bitVector = analysisInfo->getContainer(analysisInfo->_regularKillSetInfo, node->getNumber());
            if (bitVector)
               *this->_temp -= *bitVector;
-
-           bitVector = analysisInfo->getContainer(analysisInfo->_exceptionKillSetInfo, node->getNumber());
-           if (bitVector)
-              *this->_temp -= *bitVector;
+           else
+	      {
+              bitVector = analysisInfo->getContainer(analysisInfo->_exceptionKillSetInfo, node->getNumber());
+              if (bitVector)
+                 *this->_temp -= *bitVector;
+	      }
 
            bitVector = analysisInfo->getContainer(analysisInfo->_regularGenSetInfo, node->getNumber());
            if (bitVector)
               *this->_temp |= *bitVector;
+           else
```

Comment:
Another semantic difference a reason for why this is correct would be helpful.

Reply:
Same reason as stated in my response above to the first instance of semantic difference that you had asked about.